Gemfile.lock

```diff
@@ -431,6 +435,14 @@ GEM
       simplecov-html (~> 0.10.0)
     simplecov-html (0.10.0)
     single_cov (0.2.2)
+    slack-ruby-client (0.7.4)
```

Comment:
can we use https://github.com/stevenosloan/slack-notifier instead ... has 0 dependencies ...